src/pages/Embeds/Plio.vue

```diff
@@ -836,8 +907,26 @@ export default {
           if (this.isItemMCQ(itemIndex)) {
             itemResponse.answer = parseInt(itemResponse.answer);
           }
+          if (
+            !this.isItemResponseEmpty(itemIndex, itemResponse.answer) &&
+            !this.isSkipEnabled
+          ) {
+            this.lastAnsweredInteractionIndex = itemIndex;
+          }
           this.itemResponses.push(itemResponse);
         });
+        if (!this.isSkipEnabled) {
+          const firstUnansweredInteraction = this.checkMovingToTimestampAllowed(
+            this.currentTimestamp
+          );
+          // check if any item before the current timestamp is unanswered
+          // if there is, then update the timestamp to just before the
+          // unanswered item
+          if (firstUnansweredInteraction != null) {
+            this.currentTimestamp =
+              firstUnansweredInteraction.time - POP_UP_CHECKING_FREQUENCY;
+          }
+        }
```

Comment:
This logic is what is needed to be checked everytime the video is seeked
Currently you're doing it when creating session, but it should also be run whenever it is seeked. This will solve the bug that I experienced on mobile phones (added video clip in the comments on the PR)

Reply:
videoSeeked wala function mein we are calling `moveToFirstUnansweredItemTimestampOrPass` right? wahan pe ho raha same check!